R/relObsTrends.R

```diff
@@ -20,35 +20,39 @@ relObsTrends<-function(x, timeRes, focalSp){
   yearsAll <- as.numeric(dimnames(x$spatioTemporal)[[2]])
 
   spData<-deconstructOverlay(x$overlaid, attr(x, "visitCol"))
-
-  spData$group<-if(timeRes=="yearly"){
-    spData$year
+  
+  spData$dates<-as.Date(switch(timeRes,
+                     "yearly" = paste0(spData$year, "-01-01"),
+                     "monthly"= paste0(spData$year, "-", sprintf("%02d", spData$month), "-01"),
+                     "daily"  = paste0(spData$year, "-", sprintf("%02d", spData$month), "-", sprintf("%02d", spData$day))))
```

Comment:
Moved this code to the top. We can use this function to goup the observations by!